parsons/databases/db_sync.py

```diff
@@ -0,0 +1,182 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class DBSync:
+    """
+    Sync tables between databases. Works with ``Postgres`` databases.
+
+    `Args:`
+        source_db: Database connection object
+            A database object.
+        destination_db: Database connection object
+            A database object.
+        chunk_size: int
+            The number of rows per transaction copy when syncing a table. The
+            default value is 500,000 rows.
```

Comment:
For the time being, I'm going to lower this to 100K. I just want to lower the cost of each individual copy statement, even though it might take longer. It is hard to estimate the optimal size based on rows, since this doesn't take into account the number of columns.

Perhaps in the future, we could come up with some fancy estimations of total cells in a chunk, but that would be for a future PR, I think.